storage/rocksdb/ha_rocksdb.h

```diff
@@ -1000,6 +1000,8 @@ class ha_rocksdb : public my_core::handler, public blob_buffer {
   void build_decoder();
   void check_build_decoder();
 
+  inline bool is_dd_operation();
```

Comment:
Add `const`, suggesting to use "update" instead of "operation" to indicate that it involves a write: 
```suggestion
  inline bool is_dd_update() const;
```